torch/_inductor/compile_fx.py

```diff
@@ -833,7 +833,9 @@ def fw_compiler_freezing(
     # for freezing, all graph outputs should be user visible
     *_, model_outputs_node = opt_model.graph.nodes
     model_outputs = model_outputs_node.args[0]
-    user_visible_outputs = [n.name for n in model_outputs]
+    user_visible_outputs = [
+        n.name for n in model_outputs if isinstance(n, torch.fx.Node)
+    ]
```

Comment:
just curious, do we know who is using `user_visible_outputs`. I only saw assignment but no use in the code. Maybe I missed something...

Reply:
It's passed to inner_compile (should be compile_fx_inner) and eventually used in GraphLowering